test/db/formats/ar

```diff
@@ -129,150 +129,150 @@ nth paddr      vaddr      bind   type   size lib name
 nth paddr      vaddr      bind   type size lib name             
 ----------------------------------------------------------------
 1   ---------- 0x00000000 LOCAL  FILE 0        arch.c
-2   0x00000040 0x00000040 LOCAL  SECT 0        .text
```

Comment:
why these changes? What do they have to do with flirt?

Reply:
needed to properly map all the .o files inside an .a

Reply:
how is that related to flirt?

Reply:
ok, let me drop the commit on multibin and open them in a different PR